docs/exts/configuration.rst.jinja2

```diff
@@ -0,0 +1,21 @@
+{#
+ Licensed to the Apache Software Foundation (ASF) under one
+ or more contributor license agreements.  See the NOTICE file
+ distributed with this work for additional information
+ regarding copyright ownership.  The ASF licenses this file
+ to you under the Apache License, Version 2.0 (the
+ "License"); you may not use this file except in compliance
+ with the License.  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing,
+ software distributed under the License is distributed on an
+ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ KIND, either express or implied.  See the License for the
+ specific language governing permissions and limitations
+ under the License.
+#}
+{%for provider_package_name in items %}
+* :doc:`Configuration for {{ provider_package_name }} <{{ provider_package_name }}:configurations-ref>`
+{% endfor %}
```

Comment:
Wow didnt know this way possible!

Reply:
Sure. It's JINJA. You can generate anything :)